src/docs/patterns/forms.mdx

```diff
@@ -205,6 +205,8 @@ either remvoe it, or find a better way to picture this problem (like those do's
 
 # Design Specifications
 
+For additional instructions, refer to [Material Design's design specs on "Text fields"](https://material.io/components/text-fields#specs).
```

Comment:
Same question as above...do we need this here if we already link out to it in the material card above? Should we move the material card down here instead?

Reply:
I like duplication better in this case. Our persona Erice will just scroll all the way to the bottom for design specs instructions; if he can't find any, he should not be forced to scroll all the way up for that callout card, or go googling for appbar: top pattern himself.

Reply:
I agree, I'm cool with having it in two places